npm-shrinkwrap.json

```diff
@@ -12209,6 +12209,7 @@
     "node_modules/fs-xattr": {
       "version": "0.3.1",
       "dev": true,
+      "hasInstallScript": true,
```

Comment:
I have no idea if this change is necessary. I think it just happened automatically when I did `npm install`.